diesel_cli/src/cli.rs

```diff
@@ -46,12 +46,26 @@ pub fn build_cli() -> App<'static, 'static> {
         ).setting(AppSettings::SubcommandRequiredElseHelp);
 
     let setup_subcommand = SubCommand::with_name("setup")
-        .about("Creates the migrations directory, creates the database \
+        .arg(Arg::with_name("MIGRATION_DIRECTORY")
+            .long("migration-dir")
+            .help("the location of your migration directory. by default this \
+                   will look for a directory called `migrations` in the \
+                   current directory and its parents.")
+            .takes_value(true)
+            .global(true)
+        ).about("Creates the migrations directory, creates the database \
                 specified in your DATABASE_URL, and runs existing migrations.");
 
     let database_subcommand = SubCommand::with_name("database")
         .alias("db")
-        .about("A group of commands for setting up and resetting your database.")
+        .arg(Arg::with_name("MIGRATION_DIRECTORY")
+            .long("migration-dir")
+            .help("the location of your migration directory. by default this \
+                   will look for a directory called `migrations` in the \
+                   current directory and its parents.")
+            .takes_value(true)
+            .global(true)
```

Comment:
Can you

```rust
fn migration_dir_arg() -> Arg {
    Arg::with_name("MIGRATION_DIRECTORY")
        .long("migration-dir")
        .help("the location of your migration directory. by default this \
            will look for a directory called `migrations` in the \
            current directory and its parents.")
        .takes_value(true)
        .global(true)
}
```

instead of duplicating the lines?